src/ucp/core/ucp_request.inl

```diff
@@ -648,4 +648,11 @@ ucp_request_set_send_callback_param(const ucp_request_param_t *param,
     }
 }
 
+static UCS_F_ALWAYS_INLINE int
+ucp_param_is_flag(const ucp_request_param_t *param, int flag)
```

Comment:
ucp_request_param_test_flag
uint32_t flag

Reply:
i'd implement `ucp_request_param_flags`, which would return op-specific flags if `UCP_EP_PARAM_FIELD_FLAGS` is set and 0 otherwise. This looks more generic, and does not require checking `op_attr_mask` every time

Reply:
unhandled?